src/main/resources/application.yml

```diff
@@ -3,7 +3,7 @@ spring:
     name: fisioapp
 
   datasource:
-    url: jdbc:mysql://localhost:3306/fisioapp
+    url: jdbc:mysql://localhost:3306/fisioapp?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC
```

Comment:
don't change common settings unless you're adding new parameters or the change was an agreement by the team.

If you need to change the configuration define locally the parameter (you can pass via application arguments the parameter you need)